README.md

```diff
@@ -1,4 +1,12 @@
 Disclaimer: suboptimal in many ways, but more than enough for what I need.
 
+## Command line args
+
+`bs` accepts the following command line arguments:
+
+`-p`: It checkouts to the selected branch and executes a `git pull`
+    immediately afterwards.
+`-P`: It executes a `git pull` _before_ showing the branches to be selected.
```

Comment:
What's the use of the `-P` exactly? Feels kind of arbitrary

Reply:
Because it updates the branches, so if you know you want to switch to a branch you know you don't have, this will make it show up immediately. I guess `git fetch` works as well.

Reply:
I see. In that case I'd go with `git fetch` (`-F`), since `git pull` merges the current branch.